### DISCUSSION TITLE
What are categories used for?

### DISCUSSION THREAD

**Discussion by cdwilson** [INTI-CMNB/KiBot#253] [General]
I noticed that the outputs have a field called `category`, but I can't seem to find what this field us used for in the documentation (sorry if I missed it).

> `category`: [string|list(string)=''] The category for this output. If not specified an internally defined category is used. Categories looks like file system paths, i.e. PCB/fabrication/gerber.

Can someone explain how categories are used? (or point me to some description in the documentation)

**Comment by set-soft** [comment]
Hi @cdwilson !

The categories are used for the `navigate_results` output. This output can generate a web page to browse the generated files. The pages are some kind of tree and the categories are used to classify the outputs. All outputs has a default category, but you could want to use a different categorization, and this is achieved using the `category` option.

**Reply by cdwilson** [reply]
Thanks for the clarification!
